python-checks/src/test/resources/checks/hotspots/hashing/global_settings.py

```diff
@@ -1,2 +1,3 @@
-PASSWORD_HASHERS=[]  # Noncompliant
-OTHER = [] # OK
\ No newline at end of file
+PASSWORD_HASHERS=['django.contrib.auth.hashers.MD5PasswordHasher']  # Noncompliant
+PASSWORD_HASHERS=[]  # OK
+OTHER = [] # OK
```

Comment:
I don't think that the last test case is useful here. Other possible test cases:
```
PASSWORD_HASHERS=['django.contrib.auth.hashers.PBKDF2PasswordHasher']
PASSWORD_HASHERS=['django.contrib.auth.hashers.PBKDF2PasswordHasher', 'django.contrib.auth.hashers.MD5PasswordHasher']
OTHER=['django.contrib.auth.hashers.MD5PasswordHasher']
```